konfera/templates/konfera/order_details.html

```diff
@@ -0,0 +1,92 @@
+{% extends 'konfera/base.html' %}
+{% load i18n %}
+{% load custom_filters %}
+
+{% block title %}
+{% trans "Order Detail" %}
+{% endblock %}
+
+{% block content %}
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="text-center">
+          <h2>{% trans "Details for order" %} #{{ order.variable_symbol }}</h2>
+        </div>
+        <hr>
+        <div class="row">
+          <div class="col-xs-12 col-md-6">
+            <div class="panel panel-default">
+              <div class="panel-heading">{% trans "Order Details" %}</div>
+              <div class="panel-body">
+                <strong>{% trans "Event: " %}</strong>{{ event.title }}<br>
+                <strong>{% trans "Price: " %}</strong>{{ order.price|currency }}<br>
+                <strong>{% trans "Amount paid: " %}</strong>{{ order.amount_paid|currency }}<br>
+                <strong>{% trans "Discount: " %}</strong>{{ order.discount|currency }}<br>
+                <strong>{% trans "Status: " %}</strong>{{ order.status }}<br>
+                <strong>{% trans "Purchase date: " %}</strong>{{ order.purchase_date }}<br>
+                <strong>{% trans "Payment date: " %}</strong>{{ order.payment_date }}<br>
+              </div>
+            </div>
+          </div>
+          <div class="col-xs-12 col-md-6">
+            <div class="panel panel-info">
+              <div class="panel-heading">{% trans "Receipt" %}</div>
+              <div class="panel-body">
+                <strong>{% trans "Title: " %}</strong>{{ order.receipt_of.title }}<br>
+                <strong>{% trans "Amount: " %}</strong>{{ order.receipt_of.amount|currency }}<br>
+                <strong>{% trans "Street: " %}</strong>{{ order.receipt_of.street }}<br>
+                <strong>{% trans "Street2: " %}</strong>{{ order.receipt_of.street2 }}<br>
```

Comment:
I think we should show the optional fields only if they are filled in, what do you think?